src/main/java/net/dv8tion/jda/core/hooks/EventListener.java

```diff
@@ -24,7 +24,7 @@
  * <br><b>Examples: </b><br>
  * <code>
  *     JDA jda = new {@link net.dv8tion.jda.core.JDABuilder JDABuilder}(AccountType.BOT).{@link net.dv8tion.jda.core.JDABuilder#addListener(Object) addListener(listener)}.buildBlocking();<br>
- *     {@link net.dv8tion.jda.core.JDA#addEventListener(Object) jda.addEventListener(listener)};
+ *     {@link net.dv8tion.jda.core.JDA#addEventListener(Object...) jda.addEventListener(listener)};
```

Comment:
Rename the parameter here to `listeners` instead of `listener` for consistency.
You forgot to apply the same changes in the line above for **JDABuilder#addListener**